styleguide/components/31-molecules/building-blocks/quote/quote.twig

```diff
@@ -3,14 +3,13 @@
     'src': '/styleguide/img/svg/quote.svg',
     'modifier': 'quote-image',
     'alt_text': 'Quote',
```

Comment:
@delrueba Why not an empty alt? 🤔

Reply:
Good catch!
Don't use image alt text to indicate semantic meaning of neighboring elements.

Rather use correct HTML5!
q for smaller inline quotes,
blockquote for lager quotes (from another source).